platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java

```diff
@@ -427,18 +435,32 @@ public void updateForLocation(String pid, String location,
             throw new IOException("Argument configurationTable cannot be null");
         }
 
-        // sanity check to make sure no values are
-        // null
-        for (Entry<String, Object> curEntry : configurationTable.entrySet()) {
-            Object value = curEntry.getValue();
-            if (value == null || (value instanceof ArrayList && CollectionUtils.sizeIsEmpty(value))) {
-                curEntry.setValue("");
+        // find the config's metatype so we can check cardinality
```

Comment:
A comment like this is a good indicator this chunk of code should be extracted to a method if possible.